aten/src/ATen/native/NaiveConvolutionTranspose2d.cpp

```diff
@@ -281,20 +276,12 @@ void slow_conv_transpose2d_out_cpu_template(
   // Batch size + input planes
   int64_t batch_size = input_.size(0);
 
-  // Resize temporary columns
-  columns.resize_({n_output_plane * kernel_width * kernel_height,
-                   input_height * input_width});
-  columns.zero_();
+  // Create temporary columns
+  Tensor columns = at::zeros({n_output_plane * kernel_width * kernel_height,
+      input_height * input_width}, input_.options());
 
   // Define a buffer of ones, for bias accumulation
-  // Note: this buffer can be shared with other modules, it only ever gets
-  // increased, and always contains ones.
-  if (ones.dim() != 2 ||
-      ones.size(0) * ones.size(1) < output_height * output_width) {
-    // Resize plane and fill with ones...
-    ones.resize_({output_height, output_width});
-    ones.fill_(1);
-  }
+  Tensor ones = bias.defined() ? at::ones({output_height, output_width}, input_.options()) : Tensor();
```

Comment:
Hmm, is there a performance impact on this? I remember we benchmarked the slow conv finput changes, but did we benchmark this one as well?

Reply:
It looks like we were creating column and ones on the fly previously anyways, so this shouldn't have a performance impact. Is this a correct assessment?